scripts/globals/mobskills/Bubble_Shower.lua

```diff
@@ -10,6 +10,23 @@ require("scripts/globals/monstertpmoves");
 ---------------------------------------------------
 
 function onMobSkillCheck(target,mob,skill)
+	mob = mob:getID();
+	if (mob == 17129519) then

```

Comment:
That is incorrect. Mob isn't an ID, its an entity. To get a mobs ID from its entity, use `mob:getID()`. *However* we should **not** be using mob IDs as the condition for these things because mobIDs can change over time form client updates. Better to check some other condition associated with that mob, like its family ID just as an example..Actually. I think i know the best way of all:check the skill list itself because that will always be unique to the mob in question. `mob:getMobMod(MOBMOD_SKILL_LIST) == LIST_ID_HERE`

@bendangelo